apis/v1alpha1/gateway_types.go

```diff
@@ -423,6 +423,39 @@ type RouteBindingSelector struct {
 	Resource string `json:"resource"`
 }
 
+// RouteNamespaces indicate which namespaces Routes should be selected from.
+type RouteNamespaces struct {
+	// NamespaceSelector is a selector of namespaces that Routes should be
+	// selected from. This is a standard Kubernetes LabelSelector, a label query
+	// over a set of resources. The result of matchLabels and matchExpressions
+	// are ANDed. Controllers must not support Routes in namespaces outside this
+	// selector.
+	//
+	// An empty selector (default) indicates that Routes in any namespace can be
+	// selected.
+	//
+	// The OnlySameNamespace field takes precedence over this field. This
+	// selector will only take effect when OnlySameNamespace is false.
+	//
+	// Support: Core
+	//
+	// +optional
+	NamespaceSelector metav1.LabelSelector `json:"namespaceSelector"`
```

Comment:
@roberthbailey this field needs the `omitempty` tag if it's optional.

Reply:
I think you meant @robscott.

Reply:
Updated, thanks!